addons/web/static/src/core/commands/command_palette.scss

```diff
@@ -1,136 +1,38 @@
-    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-    padding: 0.8rem 1em;
-    font-size: 1.5rem;
-
-    .fa-search {
-      color:#C7C7C7;
-    }
-
-    input {
-      border: none;
-      padding: 0;
-      line-height: 1.8em;
-    }
-  }
-
-  &_listbox {
-    cursor: default;
-    max-height: 50vh;
-    overflow-y: auto;
-    position: relative;
-    scrollbar-width: none;
-    -ms-overflow-style: none;
-
-    &::-webkit-scrollbar {
-      display: none;
-    }
-
-    &_empty {
-      padding: 1rem 1.3em;
-      line-height: 1.8em;
-      font-style: italic;
-    }
-
-    .o_command_category {
-      padding: 0.5rem 0;
-    }
-
-    hr {
-      margin: 0;
+    > .modal-body {
+        padding: 0;
+        overflow: hidden;
     }
 
-    .o_command {
-      position: relative;
-      line-height: 1.8em;
-      height: 2.8em;
+    &_listbox {
+        cursor: default;
+        max-height: 50vh;
+        scrollbar-width: none;
+        -ms-overflow-style: none;
 
-      &.focused {
-        background: rgba($o-brand-primary, 0.15);
-      }
-
-      &_default, &_hotkey {
-        background-color: inherit;
-        padding: 0.5rem 1.3em;
-        display: flex;
-        align-items: center;
-        justify-content: space-between;
-      }
-
-      &_hotkey {
-        kbd {
-          background-color: #f4f7f8;
-          border-radius: 3px;
-          border: 1px solid #e5e5e5;
-          $shadowvalue: 0px 1px 1px rgba(0, 0, 0, 0.2),
-            inset 0px -1px 1px 1px rgba(230, 230, 230, 0.8),
-            inset 0px 2px 0px 0px rgba(255, 255, 255, 0.8);
-          color: #4c4c4c;
-          display: inline-block;
-          font-family: Consolas, "Liberation Mono", Courier, monospace;
-          font-size: 0.85em;
-          padding: 0 0.65rem;
+        &::-webkit-scrollbar {
+            display: none;
         }
-      }
 
-      .o_command_name {
-        > b {
-          color: $o-brand-primary;
-        }
-      }
+        .o_command {
+            &.focused {
+                background: rgba($o-brand-primary, 0.25);
+            }
 
-      .o_app_icon {
-        position: relative;
-        border-radius: 4%;
-        height: 1.8em;
-        width: 1.8em;
-
-        &> i.fa {
-          text-align: center;
-          position: absolute;
-          top: 25%;
-          left: 0;
-          bottom: auto;
-          right: 0;
+            .o_app_icon {
+                height: 1.8rem;
+                width: 1.8rem;
+            }
         }
-      }
     }
-  }
-
-  &_footer {
-    color: #707070;
-    background-color: #f6f7fa;
-    font-size: 1rem;
-    padding: 0.6rem 1.3em;
-    border-top: 1px solid rgba(0, 0, 0, 0.1);
 
-    .o_namespace:hover {
-      color: $o-brand-primary;
-      cursor: pointer;
-      text-decoration: underline dotted;
-    }
-
-    .o_promote {
-      color: $o-brand-primary;
-      font-weight: 900;
+    &_footer {
+        .o_namespace:hover {
+            color: $o-brand-primary;
```

Comment:
To achieve the same result you can combine `btn-link`, `text-primary` and `o_cursor_pointer` (not sure if the last one is needed).

Forget about the underline decoration... it's nice but not consistent.